src/components/hero/index.tsx

```diff
@@ -68,14 +66,16 @@ export const Hero = (): ReactElement => {
           {products.map((product) => {
             return (
               <SplideSlide key={product.id}>
-                <p>
-                  <h3>{product.name}</h3>
-                  {product.description.slice(0, 400)}....
-                </p>
-                <img
-                  src={`https://picsum.photos/seed/${product.name}/270`}
-                  alt="Image 1"
-                />
+                <div>
+                  <p>
+                    <h3>{product.name}</h3>
+                    {product.description.slice(0, 300)}....
+                  </p>
+                  <img
+                    src={`https://picsum.photos/seed/${product.name}/270`}
+                    alt="Image 1"
```

Comment:
Having the h3 and p together looks nice with that background! I think the h3 should be outside the p tag as its a block element - so maybe put them together in a new div, something like 
`<div>
<h3>{product.name}</h3>
<p>{product.description.slice(0, 300)}....</p>
</div>`

and then style that div

Instead of hardcoding the img alt as "Image 1" maybe use the product name?